akka-docs/rst/scala/http/client-side/websocket-support.rst

```diff
@@ -3,6 +3,61 @@
 Client-Side WebSocket Support
 =============================
 
-Not yet implemented see 17275_.
+Client side WebSocket support is available through ``Http().singleWebSocketRequest()`` ,
+``Http().webSocketClientFlow()`` and ``webSocketClientLayer``. As a WebSocket consists of two flows of
```

Comment:
the "two flows" is a bit confusing. For us flow is a very specific thing, here I guess you mean something among the lines of "a websocket connection has two sides to it, an incoming messages (a Sink) and an outgoing messages side (a Source), either of which may be signalled first, either of which can be signalled first or even "only" during the entire lifetime of an WS connection)."